shiny/_app.py

```diff
@@ -89,7 +89,7 @@ def server(input: Inputs, output: Outputs, session: Session):
     def __init__(
         self,
         ui: Tag | TagList | Callable[[Request], Tag | TagList],
-        server: Optional[Callable[[Inputs, Outputs, Session], None]],
+        server: Optional[Callable[[Inputs, Outputs, Session], None]] = None,
```

Comment:
I'm not convinced it's a good idea to have `server=None` be the default. That could save a little typing when writing tests for UI generation, but for almost all normal uses, users want a server function, and they'll benefit from an error message if it's missing.